HerPortal/wwwroot/css/app.scss

```diff
@@ -5,3 +5,72 @@
 /* Imported from Gov.UK Frontend */
 $govuk-new-link-styles: true;
 @import "govuk/all";
+
+/* Taken from DLUHC, https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/frontend/styles/_primary-navigation.scss */
```

Comment:
I think it would be a little nicer to keep this in a separate file so it's really clear what has been copied from DLUHC if were to add more CSS to the bottom of this file.